src/plotting/README.md

```diff
@@ -0,0 +1,57 @@
+Scripts for plotting data contained in either the sqlite or mysql databases.
+
+plot_barcode_family.py and plot_line_family.py are built to use sqlite as the backend. 
+
+Scripts which contain "mysql" in title reference the mysql database.
+
+
+# Usage
+
+Each script is designed to be called independently and produce a single plot. Calls are made to be to python with flags as necessary as defined below.
+
+Example call:
+```sh
+python3 plot_mysql_dir_sensor_sat.py --sidb ../satinfo 
+```
+
+## Flag options 
+
+--sidb provides the location of the satinfo folder for plots which go down to the satellite level to include the black/white list 
+if not included, then the plot will be all black and not sure any blue lines 
+
+-o the location where the plot should be saved to.
+if not included, it will save to a folder called figures in the location of the script 
+
+-dev this will append a timestamp to the name of the plot to prevent overwriting the same file during development stages 
```

Comment:
This is a nice feature! Consider making it the default behavior?

Reply:
I actually used it as the default for awhile, however, not having standard naming conventions removes our ability to work with the plots automatically. I believe it makes more sense to have the default be a single name and add the datetime flag when we want it, as I don't expect many users to want old plots but rather the most recent one anyway and it'll simplify our production calls. Do you think it should be the other way around and have some kind of prod flag?

Reply:
I usually like erring on the side of preventing overwrite. And sometimes it's nice to have a sequence of pngs to make movies.